server/rest/src/main/java/org/hawkular/apm/server/rest/AnalyticsHandler.java

```diff
@@ -61,6 +62,7 @@
 @Path("analytics")
 @Produces(APPLICATION_JSON)
 @Api(value = "analytics", description = "Analytics")
+@TenantRequired(false)
```

Comment:
It's more subtle than that, and the reason I wasn't able to easily figure out what was wrong.

Basically, other Hawkular components make use of the Tenant Filter, and the Status endpoint makes use of this annotation to mark it as `@TenantRequired(false)`. The problem is that we didn't have this tenant filter before, and now we have. This filter applies a restriction to all JAX-RS endpoints, requiring the tenant header to be provided. So, bringing the `/status` endpoint was also bringing this filter, which was making incoming requests without a `Hawkular-Tenant` to fail with a 400.

I was about to create a JIRA for this, to discuss it in more details.